internal/pkg/v2/dto/common/base/request.go

```diff
@@ -0,0 +1,13 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package base
+
+// Request defines the Request Content for request DTOs. This object and its properties correspond to the BaseRequest
+// object in the APIv2 specification.
+type Request struct {
+	CorrelationID string `json:"correlationId"`
+	RequestID     string `json:"requestId"`
+}
```

Comment:
We plan to make the model in edgex-go during experience period to avoid impacting the V1 development.  Those packages will be moved to `go-mod-core-contracts` once we confirm the approach of implementing V2 API.  We can discuss in the WG meeting.

Reply:
Discussed Core WG 9-Apr-2020 -- DTOs will be defined from the start in go-mod-core-contracts module